netsetup/connman.py

```diff
@@ -96,3 +97,28 @@ def on_services_changed(currentServices, rmServices):
 
         wifi_tech.Scan(reply_handler=lambda: logging.info('Scan complete'),
                        error_handler=lambda err: logging.error(err))
+
+    def __get_wifi_service_path(self, name):
+        if not self.manager:
+            logging.error('Unable to get wifi technology. Connman not running')
+            return
+
+        for obj_path, properties in self.manager.GetServices():
+                if properties.get('Name') == name:
+                        return obj_path
+
+    def __do_connect(self, path):
+        logging.info(path)
+        service = dbus.Interface(
+            self.bus.get_object(CONNMAN_SERVICE_NAME, path),
+            CONNMAN_SERVICE_INTERFACE)
+
+        service.Connect(reply_handler=lambda: logging.info('Connected'),
+                        error_handler=lambda err: logging.error(err))
+
+    def connect_wifi(self, name, password):
```

Comment:
This is redundant with `__do_connect()`
Shouldn't this be something like `scan_and_connect_wifi()`?